scaleiopy/im.py

```diff
@@ -85,8 +85,38 @@ def __init__(self, api_url, username, password, verify_ssl=False, LiaPassword=No
         requests.packages.urllib3.disable_warnings() # Disable unverified connection warning.
         self._cluster_config_cached = None
         self._cache_contains_uncommitted = None
+        logging.basicConfig(format='%(asctime)s: %(levelname)s %(module)s:%(funcName)s | %(message)s',
+            level=self._get_log_level(debugLevel))
+        self.logger = logging.getLogger(__name__)
+        self.logger.debug("Logger initialized!")
         
         self.SIO_Configuration_Object = None # Holds a DICT representation of the ScaleIO System Configuration
+        
```

Comment:
instead, its probably best to break functions like these into a utils module, so im.py and scaleio.py and import utils and call get log level, instead of adding repeated code.